prado/src/main/java/com/schibsted/spain/prado/extensions/Extensions.kt

```diff
@@ -28,6 +29,11 @@ fun Context.buildPradoGalleryIntent(imageUrls: List<String>,
       .putExtra(PradoGalleryActivity.EXTRA_IMAGE_PROVIDER, imageProviderType.name)
 }
 
+@JvmOverloads
```

Comment:
Q: What's the reason for using the annotation here? Does it make much of a difference without it?